src/lib/connectors/utils/requester.ts

```diff
@@ -28,11 +28,11 @@ const defaults = {
         'Cache-Control': 'no-cache',
         DNT: 1,
         Pragma: 'no-cache',
-        'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36'
+        'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/62.0.2924.87 Safari/537.36'
```

Comment:
Is there a way to remove the need to update this?

Reply:
Maybe we could add something in the release script? Also I think we should add another token for sonar with the current version. Something like `sonarwhal/0.14.1` or similar.

Reply:
Updated the list of tasks of #476

Reply:
> Maybe we could add something in the release script?

Sure.

> Also I think we should add another token for sonar with the current version

I don't know what you are referring to?

Reply:
> I don't know what you are referring to?

Add `sonarwhal/0.14.1` to the user agent string

Reply:
> Add sonarwhal/0.14.1 to the user agent string

Why do you want to do that?

Reply:
Because we should probably identify ourselves somehow?

Reply:
Would that create problems at some point because of sniffing?

Personally I think that, where we don't want to check if servers are sniffing, we shouldn't go with the default so to get the most "accurate" results as possible.

Reply:
OK